### DISCUSSION TITLE
Build Error:[Environment:fysetc_e4 error_log:sd_ESP32.cpp:324:30: error: 'class CardReader' has no member named 'diskIODriver']

### DISCUSSION THREAD

**Discussion by RockONthefeel** [luc-github/ESP3DLib#35] [Q&A] [answered]
Is there a version compatible problem or anything else?

**Comment by luc-github** [comment]
What Marlin version are you using ?

**Reply by RockONthefeel** [reply]
Marlin 2.0 Bugfix Branch based on fysetc_e4

**Comment by RockONthefeel** [comment]
static Sd2Card sdcard;
sdcard = card.getSd2Card();
if (!sd_volume.init(&sdcard)) {
    return false;
}
--------------------------------
may this can solve the problem?

**Reply by luc-github** [reply, marked as answer]
no your issue is related to Marlin version
https://github.com/luc-github/ESP3DLib/commit/40774afde7f4d320f8268aeecaa8d2687891b59c

**Reply by luc-github** [reply]
so looks like you  use old version, in that case specifiy branch 2.0.7 of esp3dlib in platformio

**Reply by RockONthefeel** [reply]
ok, thanks！
